token-lending/program/src/processor.rs

```diff
@@ -449,6 +449,8 @@ fn process_deposit_reserve_liquidity(
         clock,
         token_program_id,
     )?;
+    let log_message = format!("[4, {}]", liquidity_amount);
```

Comment:
I assume the units will be the same as the ones we get in the token summary? It technically doesn’t matter here because we only care about proportions. but just wanted to point out a reminder we are gonna be consistent of passing ints. Tokens have as many sig figs as configured by the token mint, 18 sig fig wads for fractions, etc. for future things, we passed via logging

Reply:
So this doiesn't include the decimals at all actually. It's the int representation of the decimal. We'll have to apply the decimals in the indexer. I decided to do this to save compute in the program.
Does that sound reasonable?

Reply:
Yeah that’s what i mean. Always int representation but with implicit amount of decimals. So for example 1 sol is 1000000000. 0.5 would be like 5 x 10 ^17. So yeah this works naturally. Just pointing out for if/when we meed to pass a decimal